x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -23,12 +23,19 @@ export const AGENT_NAMES: AgentName[] = [
   'python',
   'ruby',
   'rum-js',
+  'otlp',
+  'opentelemetry/cpp',
+  'opentelemetry/dotnet',
+  'opentelemetry/erlang',
+  'opentelemetry/go',
+  'opentelemetry/java',
+  'opentelemetry/nodejs',
+  'opentelemetry/php',
+  'opentelemetry/python',
+  'opentelemetry/ruby',
+  'opentelemetry/webjs',
 ];
 
-export function isAgentName(agentName: string): agentName is AgentName {
-  return AGENT_NAMES.includes(agentName as AgentName);
-}
-
 export const RUM_AGENTS = ['js-base', 'rum-js'];
```

Comment:
Not sure about this. For now I'm keeping it out because I'm not sure if all of the same data (especially all the UX stuff) is supported. I need to get a webjs app running to see what it would look ilke.